front/app/modules/commercial/customizable_navbar/admin/containers/index.tsx

```diff
@@ -1,20 +1,32 @@
 import React from 'react';
-import TabbedResource from 'components/admin/TabbedResource';
+
+// components
+import SectionFormWrapper from 'containers/Admin/pagesAndMenu/components/SectionFormWrapper';
+import Button from 'components/UI/Button';
 import { Outlet as RouterOutlet } from 'react-router-dom';
+
+// i18n
 import { InjectedIntlProps } from 'react-intl';
 import { injectIntl } from 'utils/cl-intl';
 import messages from './messages';
 
 const PagesMenu = ({ intl: { formatMessage } }: InjectedIntlProps) => {
   return (
-    <TabbedResource
-      resource={{
-        title: formatMessage(messages.pageHeader),
-        subtitle: formatMessage(messages.pageSubtitle),
```

Comment:
Don't you think there's some value in the `pageSubtitle` hint? If not, shouldn't we remove it from `front/app/modules/commercial/customizable_navbar/admin/containers/messages.ts`?

Reply:
Yeah, good catch. Will add it back.